src/main/java/com/dtolabs/rundeck/plugin/resources/ec2/EC2ResourceModelSourceFactory.java

```diff
@@ -207,6 +208,9 @@ public boolean isValid(final String s) throws ValidationException {
             .property(PropertyUtil.integer(MAX_RESULTS, "Max API Results",
                     "Max number of reservations returned per AWS API call.",
                     false, "100"))
+            .property(PropertyUtil.bool(QUERY_NODE_INSTANCES_IN_PARALLEL, "Query Node Instances in Parallel",
+                    "Query node instances in parallel. If false, instances will be queried one at a time.",
+                    false, "false"))
```

Comment:
any reason not to set this to true by default?

Reply:
Only reason would be to not change the behavior for existent configurations.

Reply:
ok let's leave it false for now, maybe in a future version we can make it true by default